torch/onnx/utils.py

```diff
@@ -694,7 +694,7 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
         if opset_version is None:
             opset_version = _default_onnx_opset_version
         if not operator_export_type:
```

Comment:
Set the default in the function signature to `OperatorExportTypes.ONNX and remove this logic.